rock_scissor_paper_assignment.js

```diff
@@ -1 +1,20 @@
-// code here
+var userChoice=prompt("Do you choose rock ,paper or scissors?");
+var computerChoice=Math.random();
+
+console.log(computerChoice);
+if (computerChoice<=0.33) {
+    computerChoice="rock";
+} else if (0.34<=computerChoice<=0.66) {
+    computerChoice="scissors";
+} else {
+    computerChoice="paper";
+}
+console.log(computerChoice);
+console.log(userChoice);
+if (userChoice === "paper" && computerChoice ==="scissors" || userChoice === "scissors" && computerChoice==="rock" || userChoice === "rock" && computerChoice === "paper") {

```

Comment:
nice, no nested if :). minor thing, this line is too long, what about putting each `||` to new line?